### DISCUSSION TITLE
[Feature Request] Rotating shuffle

### DISCUSSION THREAD

**Discussion by 3hhh** [qtile/qtile#5072] [Ideas]
Currently shuffling windows via Mod+shift+j/k in e.g. the Monad layout stops once there is no further left/right/up/down.

It would be nice to be able to continuously press e.g. Mod+shift+j and always cause a shuffle, starting again from the beginning once the end is reached.

**Comment by elParaguayo** [comment]
You can do `lazy.group.next_window()` which should cycle (I think).

**Reply by 3hhh** [reply]
Actually `lazy.layout.down()` in MonadTall cycles as well and I'm happy with that.

However `lazy.layout.shuffle_down()` doesn't cycle (I was only talking about the shuffle, not about the window focus).

**Reply by elParaguayo** [reply]
Sorry. Misunderstood.

**Reply by 3hhh** [reply]
No worries!
